Description: do not ship extragear-network in kde-l10n-sr
Author: Modestas Vainius <modax@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=584633

--- a/sr/sr@ijekavian/messages/CMakeLists.txt
+++ b/sr/sr@ijekavian/messages/CMakeLists.txt
@@ -1,4 +1,3 @@
-add_subdirectory(extragear-network)
 add_subdirectory(kdeaccessibility)
 add_subdirectory(kdeadmin)
 add_subdirectory(kdeartwork)
--- a/sr/sr@ijekavianlatin/messages/CMakeLists.txt
+++ b/sr/sr@ijekavianlatin/messages/CMakeLists.txt
@@ -1,4 +1,3 @@
-add_subdirectory(extragear-network)
 add_subdirectory(kdeaccessibility)
 add_subdirectory(kdeadmin)
 add_subdirectory(kdeartwork)
--- a/sr/sr@latin/messages/CMakeLists.txt
+++ b/sr/sr@latin/messages/CMakeLists.txt
@@ -1,4 +1,3 @@
-add_subdirectory(extragear-network)
 add_subdirectory(kdeaccessibility)
 add_subdirectory(kdeadmin)
 add_subdirectory(kdeartwork)
