--- a/engb/docs/kdepim/kaddressbook/CMakeLists.txt
+++ b/engb/docs/kdepim/kaddressbook/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/nn/docs/kdepim/kaddressbook/CMakeLists.txt
+++ b/nn/docs/kdepim/kaddressbook/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/pt/docs/kdepim/kabcclient/man-kabcclient.1.docbook
+++ b/pt/docs/kdepim/kabcclient/man-kabcclient.1.docbook
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY % Portuguese "INCLUDE"
 ><!-- change language only here -->
 ]>
@@ -9,10 +9,10 @@
 <title
 >Manual do Utilizador do &kde;</title>
 <date
->2005-09-08</date
-> <productname
->Ambiente de Trabalho K</productname
-> </refentryinfo>
+>2005-09-08</date>
+<productname
+>Ambiente de Trabalho K</productname>
+</refentryinfo>
 
 <refmeta>
 <refentrytitle
--- a/fr/docs/kdepim/kalarm/CMakeLists.txt
+++ b/fr/docs/kdepim/kalarm/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/fr/docs/kdepim/konsolekalendar/CMakeLists.txt
+++ b/fr/docs/kdepim/konsolekalendar/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/fr/docs/kdepim/korganizer/CMakeLists.txt
+++ b/fr/docs/kdepim/korganizer/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/uk/docs/kdepim/kabcclient/man-kabcclient.1.docbook
+++ b/uk/docs/kdepim/kabcclient/man-kabcclient.1.docbook
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY % Ukrainian "INCLUDE"
 ><!-- change language only here -->
 ]>
--- a/hu/docs/kdepim/kaddressbook/CMakeLists.txt
+++ b/hu/docs/kdepim/kaddressbook/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/hu/docs/kdepim/kcontrol/kalarmd/CMakeLists.txt
+++ b/hu/docs/kdepim/kcontrol/kalarmd/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/ SUBDIR kcontrol/kalarmd )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/ SUBDIR kcontrol/kalarmd )
--- a/hu/docs/kdepim/kjots/CMakeLists.txt
+++ b/hu/docs/kdepim/kjots/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/hu/docs/kdepim/knotes/CMakeLists.txt
+++ b/hu/docs/kdepim/knotes/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/sl/docs/kdepim/kaddressbook/CMakeLists.txt
+++ b/sl/docs/kdepim/kaddressbook/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
+#kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/${CURRENT_LANG}/  )
--- a/it/docs/kdepim/kabcclient/man-kabcclient.1.docbook
+++ b/it/docs/kdepim/kabcclient/man-kabcclient.1.docbook
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY % Italian "INCLUDE"
 ><!-- change language only here -->
 ]>
--- a/sv/docs/kdepim/kabcclient/man-kabcclient.1.docbook
+++ b/sv/docs/kdepim/kabcclient/man-kabcclient.1.docbook
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY % Swedish "INCLUDE"
 ><!-- change language only here -->
 ]>
